.github/workflows/docker-release.yml

```diff
@@ -1,58 +1,98 @@
+          docker push $IMAGE1:${{ github.ref_name }}-${{ matrix.tag_suffix }}
+
+      - name: Build and push juspay-mcp for sse
+        run: |
+          nix run .#docker-sse.copyToDockerDaemon --system ${{ matrix.system }}
+          docker tag juspay-mcp-sse:latest $IMAGE1:sse-${{ github.ref_name }}-${{ matrix.tag_suffix }}
+          docker push $IMAGE1:sse-${{ github.ref_name }}-${{ matrix.tag_suffix }}
+
+      - name: Build and push juspay-dashboard-mcp
+        run: |
+          nix run .#docker-dashboard.copyToDockerDaemon --system ${{ matrix.system }}
+          docker tag juspay-dashboard-mcp:latest $IMAGE2:${{ github.ref_name }}-${{ matrix.tag_suffix }}
+          docker push $IMAGE2:${{ github.ref_name }}-${{ matrix.tag_suffix }}
+
+      - name: Build and push juspay-dashboard-mcp for sse
+        run: |
+          nix run .#docker-dashboard-sse.copyToDockerDaemon --system ${{ matrix.system }}
+          docker tag juspay-dashboard-mcp-sse:latest $IMAGE2:sse-${{ github.ref_name }}-${{ matrix.tag_suffix }}
+          docker push $IMAGE2:sse-${{ github.ref_name }}-${{ matrix.tag_suffix }}
+
+  manifest:
+    needs: build-and-push
+    runs-on: ubuntu-latest
+    env:
+      IMAGE1: ${{ secrets.DOCKERHUB_NAMESPACE }}/juspay-mcp
+      IMAGE2: ${{ secrets.DOCKERHUB_NAMESPACE }}/juspay-dashboard-mcp
+
+    steps:
+      - name: 🔐 Log in to Docker Hub
+        uses: docker/login-action@v3
         with:
-          context: .
-          file: Dockerfile.sse
-          platforms: linux/amd64,linux/arm64
-          push: true
-          tags: |
-            ${{ secrets.DOCKERHUB_NAMESPACE }}/juspay-mcp:sse-latest
-            ${{ secrets.DOCKERHUB_NAMESPACE }}/juspay-mcp:sse-${{ github.ref_name }}
+          username: ${{ secrets.DOCKERHUB_USERNAME }}
+          password: ${{ secrets.DOCKERHUB_PASSWORD }}
+
+      - name: Create and push multi-arch manifest for juspay-mcp and sse
+        run: |
+          docker manifest create $IMAGE1:latest \
+            --amend $IMAGE1:${{ github.ref_name }}-amd64 \
+            --amend $IMAGE1:${{ github.ref_name }}-arm64
+          docker manifest push $IMAGE1:latest
+          docker manifest create $IMAGE1:sse-latest \
+            --amend $IMAGE1:sse-${{ github.ref_name }}-amd64 \
+            --amend $IMAGE1:sse-${{ github.ref_name }}-arm64
+          docker manifest push $IMAGE1:sse-latest
+
+      - name: Create and push multi-arch manifest for juspay-dashboard-mcp and sse
+        run: |
+          docker manifest create $IMAGE2:latest \
+            --amend $IMAGE2:${{ github.ref_name }}-amd64 \
+            --amend $IMAGE2:${{ github.ref_name }}-arm64
+          docker manifest push $IMAGE2:latest
+          docker manifest create $IMAGE2:sse-latest \
+            --amend $IMAGE2:${{ github.ref_name }}-amd64 \
+            --amend $IMAGE2:${{ github.ref_name }}-arm64
+          docker manifest push $IMAGE2:sse-latest
```

Comment:
Can't you use a bash function or something to respect [DRY](https://en.wikipedia.org/wiki/Don%27t_repeat_yourself) here?